README.md

```diff
@@ -131,13 +131,9 @@ examples of how to make this call.
 
 
 ## More information/Support
-For more information and support, refer to the competition guidelines which you should have received when registering
-for the contest, or post your questions in the support Slack channel, to which you should have also gained access 
-upon registering.
+For more information and support, refer to the competition guidelines or post your questions in the support Slack channel, both of which you should have gained access when you create a login in the competition platform.
 
-An example dashboard using Cognizant-trained predictors and prescriptors can be found 
-[here](https://evolution.ml/demos/npidashboard/). Using this dashboard you can select among different prescriptors
-from the Pareto Front to see the effect on prescriptions for intervention plans in various regions.
+For a concrete visualization of what the competition is about, see Cognizant's [COVID-19 intervention optimization demo](https://evolution.ml/demos/npidashboard/). Using this dashboard you can select among different prescriptors from the Pareto Front to see the effect on prescriptions for intervention plans in various regions.
 
 For more background information please see also the research paper 
```

Comment:
I was a bit torn here about whether to link directly to the paper, -or- to the page on `evolution.ml` which discusses the paper and then links to the paper. Thoughts?

Reply:
Yes I've struggled with that too, and also whether to point people to the demo directly or to the esp/npi page. Usually I've used the esp/npi, but since there's so much already about the npis in the Xprize competition, I'm thinking people would rather go directly to the demo and papers. But either way is fine by me.